### PR TITLE
Summary on page

### PR DESCRIPTION
**Pull request by GeorgeGorbanev**

### CODE REVIEW

File: src/js/url/stripper.js [hunk 1]
```diff
@@ -11,14 +11,14 @@ const URL_REGEXP = new RegExp([
...
   return match && {
-        protocol: match[2],
-        host: match[3],
-        hostname: match[4],
-        port: match[5],
-        pathname: match[6],
-        search: match[7],
-        hash: match[8]
-      };
+    protocol: match[2],
```
Comment (zeckson) [inline src/js/url/stripper.js]: 👍

File: src/js/summary.js [hunk 2]
```diff
@@ -0,0 +1,9 @@
+export default {
+  update(newSummary) {
+    document.getElementById(`password-length`).innerHTML = newSummary.password.length;
```
Comment (zeckson) [inline src/js/summary.js]: Let's move all DOM selectors to top of module. To not recalculate them on every `update`

Comment (GeorgeGorbanev) [inline src/js/summary.js]: Done

**Comment by GeorgeGorbanev** [comment]
Now pass textHash() to vernam.encrypt()

**Comment by GeorgeGorbanev** [comment]
Created util.js, textHash() function and simple test

**Pull request merged**